lib/Doctrine/Migrations/Tools/Console/Command/GenerateCommand.php

```diff
@@ -53,6 +53,16 @@ public function execute(InputInterface $input, OutputInterface $output) : void
         }
 
         $output->writeln(sprintf('Generated new migration class to "<info>%s</info>"', $path));
+
+        $output->writeln(sprintf(
+            "\nTo run just this migration for testing purposes, you can use <info>migrations:execute --up %s</info>",
+            $versionNumber
+        ));
+
+        $output->writeln(sprintf(
```

Comment:
Instead of multiple consecutive calls to `writeln()`, and instead of prefixing messages with `"\n"`, what do you think about passing an array of message strings (and empty strings for new lines) to `writeln()` instead?

Reply:
Changed.